promisasu/Portal
# Update Logging

**Taiga User Story**
None

**Taiga Task(s)**
None

**What did you change?**
Updated `good` logger to latest version.
Separated request logging in a different file from ops logging.

**How can we test?**
`npm install`
`npm start`
check the log files there should be:
```
prp-development-api-error.log        prp-development-dashboard-out.log    prp-development-api-access.log        prp-development-dashboard-ops.log
prp-development-api-out.log          prp-development-scheduler-error.log  prp-development-api-ops.log
prp-development-dashboard-error.log  prp-development-scheduler-out.log    prp-development-dashboard-access.log
```
